refgenconf/const.py

```diff
@@ -10,6 +10,7 @@
 CFG_CONST = ["CFG_NAME", "CFG_ENV_VARS"]
 DEFAULT_SERVER = "http://refgenomes.databio.org"
 API_VERSION = "v3"
+OLD_API_VERSION = "v2"
```

Comment:
let's change the var name to `API_VERSION_2` since next time `"v3"` will become `OLD*` and this `OLDER*`?

Reply:
5f7f4c1